corehq/apps/custom_data_fields/edit_model.py

```diff
@@ -35,6 +35,7 @@ class CustomDataFieldsForm(forms.Form):
     data_fields = forms.CharField(widget=forms.HiddenInput)
     purge_existing = forms.BooleanField(widget=forms.HiddenInput, required=False, initial=False)
     profiles = forms.CharField(widget=forms.HiddenInput)
+    req_profile = forms.CharField(widget=forms.HiddenInput, required=False)
```

Comment:
nit: would be clearer to just spell it all out? From here, I would expect this to be short for `requires_profile` and expect a boolean field.

Reply:
I agree it would be clearer to spell out 'require'. 
With the current functionality this will hold a string representing which user types require a profile selection: 
'web_user,commcare_user', 'web_user', 'commcare_user' or 'neither'.